cli/register/register.go

```diff
@@ -629,6 +629,17 @@ func verifyRegisterParamters(org, pattern, nodeOrgFromFlag string, patternFromFl
 		}
 	}
 
+	if nodeId, _ := cliutils.SplitIdToken(nodeIdTok); nodeId != "" {
+		illegalInput, err := api.InputIsIllegal(nodeId)
+		if err != nil {
+			cliutils.Fatal(cliutils.CLI_GENERAL_ERROR, msgPrinter.Sprintf("could not validate node ID '%s': %v", nodeId, err))
+		} else if illegalInput != "" {
+			cliutils.Fatal(cliutils.CLI_INPUT_ERROR, msgPrinter.Sprintf("invalid node ID: A-Za-z0-9 and special symbols !-*+()?.,:&@ are only allowed"))
```

Comment:
"invalid node ID: A-Za-z0-9, unicode characters and special symbols !-*+()?.,:&@ are only allowed"